Roys/.idea/misc.xml

```diff
@@ -79,8 +45,9 @@
   </component>
   <component name="masterDetails">
     <states>
-      <state key="ScopeChooserConfigurable.UI">
+      <state key="ProjectJDKs.UI">
         <settings>
+          <last-edited>1.7</last-edited>
```

Comment:
I removed jdk1.7 version and it works and also app works well even if I removed that line .
But when I push after remove that line git said everything is up-to-date. I don't know the reasonㅠㅠ